tests/test_enterprise/api/test_views.py

```diff
@@ -629,28 +688,49 @@ def test_enterprise_customer_catalogs_list(self, is_staff, is_linked_to_enterpri
         assert response == expected_results
 
     @ddt.data(
-        (False, False, {'detail': 'Not found.'}),
+        (
+            False,
+            False,
+            False,
+            {'detail': 'Not found.'},
+        ),
+        (
+            False,
+            False,
+            True,
+            {'detail': 'Not found.'},
+        ),
         (
             False,
             True,
+            True,
             fake_enterprise_api.build_fake_enterprise_catalog_detail(include_enterprise_context=True),
         ),
         (
             True,
             False,
+            True,
             fake_enterprise_api.build_fake_enterprise_catalog_detail(include_enterprise_context=True),
         ),
         (
             True,
             True,
+            True,
             fake_enterprise_api.build_fake_enterprise_catalog_detail(include_enterprise_context=True),
         ),
     )
```

Comment:
@UmanShahzad There's a few combinations missing here:

* `is_staff`: `False`, `is_linked_to_enterprise`: `True`, `has_permissions`: `False`
* `is_staff`: `True`, `is_linked_to_enterprise`: `True`, `has_permissions`: `False`
* `is_staff`: `True`, `is_linked_to_enterprise`: `False`, `has_permissions`: `False`

Would any of these make sense to test, in addition to the cases that are already covered?

Reply:
Sure, I'll add them and take a look! :+1:

Reply:
So yeah, as below, turned out this was obsolete :P